File: app/src/main/java/link/couple/jin/couplelink/fcm/CpFirebaseIDService.java
package link.couple.jin.couplelink.fcm;

import com.google.firebase.iid.FirebaseInstanceId;
import com.google.firebase.iid.FirebaseInstanceIdService;

import link.couple.jin.couplelink.utile.Log;

/**
 * Created by jeongjin-yong on 2017. 9. 20..
 */

public class CpFirebaseIDService extends FirebaseInstanceIdService {

    @Override
    public void onTokenRefresh() {
        String refreshedToken = FirebaseInstanceId.getInstance().getToken();
        Log.d("Refreshed token: " + refreshedToken);
    }
}

File: app/src/main/java/link/couple/jin/couplelink/fcm/CpFirebaseIDService.kt
package link.couple.jin.couplelink.fcm

import com.google.firebase.iid.FirebaseInstanceId
import com.google.firebase.iid.FirebaseInstanceIdService

import link.couple.jin.couplelink.utile.Log

/**
 * Created by jeongjin-yong on 2017. 9. 20..
 */

class CpFirebaseIDService : FirebaseInstanceIdService() {

    override fun onTokenRefresh() {
        val refreshedToken = FirebaseInstanceId.getInstance().token
        Log.d("Refreshed token: " + refreshedToken!!)
    }
}

